src/base/RunnerUtil.ml

```diff
@@ -192,6 +192,7 @@ type runner_cli = {
   cf_token_fields : string list;
   p_contract_info : bool;
   p_type_info : bool;
+  silence_warnings : bool;
```

Comment:
I'd suggest to rename this flag. To "silence warnings" suggests performing checks and then hiding the results, whereas here, afaiu, the checks themselves are not performed at all if the flag is `true`.

Reply:
Renamed it to `disable_analy_warn` instead! Let me know if this works :)